decidim-blogs/lib/decidim/blogs/engine.rb

```diff
@@ -14,7 +14,7 @@ class Engine < ::Rails::Engine
       isolate_namespace Decidim::Blogs
 
       routes do
-        resources :posts, only: [:index, :show]
+        resources :posts, only: [:index, :show, :new, :create, :edit, :update, :destroy]
```

Comment:
```suggestion
        resources :posts
```

resources by default contains index, show, new, edit, create, update, and destroy